src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible'], value: ' '}
+  }
+
+  handleChange(event) {
+    this.setState({value: event.state.comments});
+  }
+
+  handleClick(){
+    if(this.state.value.length > 0){
+    this.setState({comments: _.concat(this.state.comments, this.state.value)})
+    this.state.value = ''
+    }
+  }
+
+  render(){
+    
+    return (
+      <form>
+        {_.map(this.state.comments, comment => <li>{comment}</li>)}
+        <input type="text" value={this.state.comments} onChange={this.handleChange}/>
```

Comment:
The value prop here is not correct, you want the input to display the value that is being entered into the input